modules/appeals_api/lib/appeals_api/token_validation_client.rb

```diff
@@ -4,19 +4,37 @@
 require 'token_validation/v2/configuration'
 
 module AppealsApi
+  class TokenValidation
+    module V3
+      class Configuration < ::TokenValidation::V2::Configuration
+        def connection
+          Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
+            conn.request :url_encoded # required for token validation service v3 (v2 accepted json)
+            conn.response :snakecase
+            conn.adapter Faraday.default_adapter
+          end
+        end
+      end
+    end
+  end
+
   # This is based on TokenValidation::V2::Client, but allows for more fine-grained status codes in failure responses
   class TokenValidationClient < ::Common::Client::Base
-    configuration TokenValidation::V2::Configuration
+    configuration TokenValidation::V3::Configuration
 
     def initialize(api_key:)
       @api_key = api_key
     end
 
     def validate_token!(audience:, token:, scopes:)
-      json = { 'aud': audience }
-      headers = { 'apiKey': @api_key, 'Authorization': "Bearer #{token}" }
-
-      response = perform(:post, 'v3/validation', json, headers)
```

Comment:
I didn't think to update this before because the rake tasks were working fine to fetch and validate tokens against v3 (they were already using `x-www-form-urlencoded`). It doesn't look like the v3 validation service docs are published yet, but the [docs for v2](https://api.va.gov/internal/auth/docs/v2/validation.json) specify that the body should be form data.